src/modules/dashboard/DashboardTopPanel.tsx

```diff
@@ -259,6 +260,7 @@ export const DashboardTopPanel = () => {
                       symbol: symbols[0],
                       underlyingAsset: claimAddresses[0],
                       iconSymbol: symbols[0],
+                      decimals: decimals[0],
```

Comment:
Had to add decimals in, was splatting an object with 18 decimals before casting as `ComputedReserveData` @fredwes